packages/aws-cdk/lib/logging.ts

```diff
@@ -185,3 +170,60 @@ export const success = (input: LogInput<'I'>, ...args: unknown[]) => {
 export const highlight = (input: LogInput<'I'>, ...args: unknown[]) => {
   return formatMessageAndLog('info', input, chalk.bold, ...args);
 };
+
+/**
+ * Helper class to emit standard log messages to an IoHost
+ *
+ * It wraps an `IoHelper`, and adds convenience methods for the various log
+ * levels, same as the global ones but scoped to a particular `IoHost` instead
+ * of the global singleton one.
+ */
+export class IoLogger {
```

Comment:
So now we have two sets of deprecated logging functions?

Reply:
I don't like anything that's permanent to be called logging. It gives a false impression to what it's doing. So maybe this?

```suggestion
export class IoMessageEmitter {
```

Reply:
Ha, 100% what I had first and then I changed it :D